CompanyEmployees.Infrastructure.Persistence/Repositories/AttendanceRepository.cs

```diff
@@ -0,0 +1,41 @@
+using CompanyEmployees.Core.Domain.Entities;
+using CompanyEmployees.Core.Domain.Repositories;
```

Comment:
The file-level namespace declaration uses a semicolon format (`namespace ...;`) instead of block style, which may impact readability or consistency depending on the rest of the project. Consider reviewing whether the semicolon-style namespace matches global coding conventions for this project.

**Suggested change**
```suggestion
namespace CompanyEmployees.Infrastructure.Persistence.Repositories
{
```